api/CSSPropertyRule.json

```diff
@@ -11,7 +11,7 @@
             "version_added": "78"
           },
           "edge": {
-            "version_added": "≤79"
+            "version_added": "85"
```

Comment:
It's surprising if this was enabled in Chrome 78 but not until Edge 85. Is the Chrome data incorrect, or what's going on?

Reply:
The answer is that https://mdn-bcd-collector.appspot.com/tests/api/CSSPropertyRule also starts returning true in Chrome 85.

Reply:
OK, I've commented in https://github.com/mdn/browser-compat-data/pull/6278#issuecomment-782663612. I'll merge this PR and send another to fix the rest.